py/selenium/webdriver/common/action_chains.py

```diff
@@ -235,13 +235,13 @@ def move_to_element(self, to_element):
         return self
 
     def move_to_element_with_offset(self, to_element, xoffset, yoffset):
-        """Move the mouse by an offset of the specified element. Offsets are
-        relative to the top-left corner of the element.
+        Move the mouse by an offset of the specified element. Offsets are
```

Comment:
accidental removal of the docstring prefix here

Reply:
Thank you. I just fixed it.